### PR TITLE
Fixes defenders description

### PR DESCRIPTION
**Pull request by DummeHytteOst**
## About the PR
changes "A alien", to "An alien" in the defenders description

its also an issue in CM13, if anyone wanna bother making a fix for them 🔨 
## Requirements

- [x] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
:cl:
- fix: Corrected the defenders description

### CODE REVIEW

File: Resources/Prototypes/_RMC14/Entities/Mobs/Xeno/defender.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
-﻿- type: entity
+- type: entity
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Entities/Mobs/Xeno/defender.yml:1]: I think your editor changed the encoding of the file accidentally judging by the modified first line
```suggestion
- type: entity
```

**Review by DrSmugleaf — APPROVED** [review]
Thanks for the PR!

File: Resources/Prototypes/_RMC14/Entities/Mobs/Xeno/defender.yml (hunk 1 above)
Comment (DummeHytteOst) [reply Resources/Prototypes/_RMC14/Entities/Mobs/Xeno/defender.yml:1]: don't know how i can fix it, doesn't seem to actually change anything, might just be git being git, should be good tho

**Comment by DummeHytteOst** [comment]
welp, don't know why it says the first line is changed, might just be git, nothing actually changed and it works so ¯\_(ツ)_/¯

**Pull request merged**